jjb/dynamic/uperf.yml

```diff
@@ -0,0 +1,101 @@
+- job:
+    block-downstream: false
+    block-upstream: false
+    builders:
+    - shell: |+
+        set -o pipefail
+        set -eux
+        # Disable logging
+        set +x
+        oc login -u ${CLUSTER_USER} -p ${CLUSTER_PASSWORD} ${CLUSTER_API_URL}
+        # Run tests
+        set -x
+        pushd plow/workloads/network-perf/
```

Comment:
This github jenkins plugin clones the content in the workspace directory without the repo name, so this needs to "pushd workloads/network-perf"